src/screens/WalletScreen.js

```diff
@@ -103,7 +146,15 @@ export default function WalletScreen() {
         setFetchedCharts(true);
       }
     }
-  }, [assetsSocket, dispatch, fetchedCharts, initialized, sections]);
+  }, [
+    assetsSocket,
+    dispatch,
+    fetchedCharts,
+    initialized,
+    sections,
+    trackPortfolios,
```

Comment:
do you need `trackPortfolios` and `userAccounts` in this particular useEffect's dependency list?